### PR TITLE
Workaround for DiskFilter

### PR DESCRIPTION
**Pull request by Xiaohua-Shen**
Before community provide https://bugs.launchpad.net/nova/+bug/1593155
related fix, we need this workaround to enable disk filter.

### CODE REVIEW

File: roles/nova-common/templates/custom-filters/custom_disk_filter.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2012 OpenStack Foundation
```
Comment (nirajdp76) [inline roles/nova-common/templates/custom-filters/custom_disk_filter.py]: ues should make a phyton package - bbc-openstack-plugin .. We do something similar thing for glance. I will work with @cmt to port this to it's own pthon package.

**Comment by nirajdp76** [comment]
@craigtracey can help look at this.

File: roles/nova-common/templates/custom-filters/custom_disk_filter.py (hunk 1 above)
Comment (Xiaohua-Shen) [inline roles/nova-common/templates/custom-filters/custom_disk_filter.py]: I found bbc-openstack-plugins repo , will move it to that repo

Comment (nirajdp76) [inline roles/nova-common/templates/custom-filters/custom_disk_filter.py]: shouldn;t this file be deleted from this PR given is now part of bbc-openstack-plugins

**Comment by nirajdp76** [comment]
retest

**Comment by nirajdp76** [comment]
retest

**Pull request merged**